corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -206,10 +207,13 @@ def __record_change_metric_in_datadog(self, metric, change, timer=None):
                 u'document_type:{}'.format(change.metadata.document_type),
                 u'domain:{}'.format(change.metadata.domain),
                 u'is_deletion:{}'.format(change.metadata.is_deletion),
-                u'pillow_name:{}'.format(self.get_name())
+                u'pillow_name:{}'.format(self.get_name()),
             ]
             datadog_counter(metric, tags=tags)
 
+            change_lag = (datetime.utcnow() - change.metadata.publish_timestamp).seconds
+            datadog_gauge('commcare.change_feed.change_lag', change_lag, tags=tags)
```

Comment:
I don't think all these tags make sense for this metric. I think we could limit it to:
* pillow_name
* topic

I think you may have to add the topic to the change though. You can add it here: https://github.com/dimagi/commcare-hq/blob/be5c8ce9ea6f9cf1c16894570505172911ccbb7c/corehq/apps/change_feed/consumer/feed.py#L179
```
topic = message.topic
```